src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Threading/IdeFactDiscoverer.cs

```diff
@@ -28,7 +28,10 @@ public IEnumerable<IXunitTestCase> Discover(ITestFrameworkDiscoveryOptions disco
                     foreach (var supportedInstance in GetSupportedInstances(testMethod, factAttribute))
                     {
                         yield return new IdeTestCase(_diagnosticMessageSink, discoveryOptions.MethodDisplayOrDefault(), discoveryOptions.MethodDisplayOptionsOrDefault(), testMethod, supportedInstance);
-                        yield return new IdeInstanceTestCase(_diagnosticMessageSink, discoveryOptions.MethodDisplayOrDefault(), discoveryOptions.MethodDisplayOptionsOrDefault(), CreateVisualStudioTestMethod(supportedInstance), supportedInstance);
+                        if (IdeInstanceTestCase.TryCreateNewInstanceForFramework(discoveryOptions, _diagnosticMessageSink, supportedInstance) is { } instanceTestCase)
```

Comment:
I'm not super hip to pattern matching, it looks like you were going to do property matching but then it was blank?

Reply:
`is { }` means not-null. Related patterns are:

* `is var x`: This uses type inference for `x`, but is always true (doesn't perform a null check)
* `is object x`: This ensures `x` is non-null, but doesn't use type inference to create a value of the correct type
* `is not null`: This ensures the value is not null, but you can't create a variable with the result (i.e. `is not null x` is not a valid pattern)